### PR TITLE
task completed

### PR DESCRIPTION
**Pull request by Pokhvalenkos**
- [DEMO LINK](https://Pokhvalenkos.github.io/layout_stop-watch/)

### CODE REVIEW

File: src/styles/index.scss [hunk 1]
```diff
@@ -1,3 +1,76 @@
...
+}
+
+@keyframes rotateSecondHand {
+  from {
+    transform: rotate(0deg);
+  }
+  to {
+    transform: rotate(360deg);
+  }
+}
```
Comment (nazarmatsevych) [inline src/styles/index.scss:68]: Use just one animation

**Review by nazarmatsevych — CHANGES_REQUESTED** [review]
Good work but animation doesn't work, try to fix it or ask for help in fe_chat

File: src/index.html [hunk 2]
```diff
@@ -17,6 +17,10 @@
     />
   </head>
   <body>
-    <h1>Stop watch</h1>
+    <div class="stopwatch">
```
Comment (maxim2310) [inline src/index.html:20]:
```suggestion
    <div class="stopwatch stopwatch--speed-up">
```

**Review by maxim2310 — CHANGES_REQUESTED** [review]
Yor demo link is not working, please fix it and make request review again

**Review by VitaliyBondarenko1982 — CHANGES_REQUESTED** [review]
There are some problems with minute hand.
Screenshot 2024-03-28 at 18 21 11
Please try to fix it.

**Pull request closed without merging**